### ISSUE TITLE
There should be a way to extend the usage of IMDB example

### ISSUE THREAD

**Issue by vinhqdang** [fchollet/keras#1246]
I think the examples provided with IMDB sentiment analysis is cool, but it is specifically written for imdb dataset.

I think, the idea of this example is text classification, with a set of documents with label, then use this dataset to predict the label of new document.

It would be better if there is another utilities which will read the bunch of documents and analyze the frequency of these documents, so there will be X_train, X_test automatically, and x_train, x_test can be fetched manually.

So, it may looks like:

docs = doc.load_data (path_to_directory_contains_all_documents)
label = doc.load_label (path_to_file_contains_label_of_all_documents)
docs_train = ...
docs_test = ...
...

**Comment by fchollet** [comment]
Easy to do by yourself. There are text processing utilities in Keras (and also in sklearn) that make this quick and simple.

**Comment by vinhqdang** [comment]
Yeah, you are right, I found it easily to do. Thanks a lot,
